snuba/datasets/events.py

```diff
@@ -32,6 +32,7 @@
     all_referenced_columns,
     escape_literal,
     string_col,
+    qualified_column,
```

Comment:
The changes in this file are only reinforcing my thinking that datasets shouldn't be allowed to make these kind of query changes in the future — they should come from column definitions or extensions.

Reply:
For "this kind of query changes" do you mean "everything that happens in column_expr" ?
If yes, then I agree, I wrote my theory about this topic in a previous comment (https://github.com/getsentry/snuba/pull/474#discussion_r326932220). I am planning to validate that approach in the next days.

Reply:
I think so, yes.